src/shared/components/query/QueryStore.ts

```diff
@@ -1076,26 +1078,21 @@ export class QueryStore {
     }
 
     readonly sampleLists = remoteData({
+        await: () => [
+            this.sampleListsInStudyCache.get(
+                this.selectableSelectedStudyIds[0]
```

Comment:
we should include a comment explaining why it's ok to just select first one (i think it's b/c you would only be able to select a sample list if there was only a single study selected, right?)

Reply:
this logic was already there, and I don't really know why this is ok :man_shrugging: